hathor/reactor/reactor.py

```diff
@@ -14,25 +14,55 @@
 
 from typing import cast
 
+from structlog import get_logger
 from twisted.internet.interfaces import IReactorCore, IReactorTCP, IReactorTime
 from zope.interface.verify import verifyObject
 
 from hathor.reactor.reactor_protocol import ReactorProtocol
 
+logger = get_logger()
+
 # Internal variable that should NOT be accessed directly.
 _reactor: ReactorProtocol | None = None
 
 
-def get_global_reactor() -> ReactorProtocol:
+def get_global_reactor(*, use_asyncio_reactor: bool | None = None) -> ReactorProtocol:
```

Comment:
Maybe create an `initialize_global_reactor()` that must be called first.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/889/commits/70f2ad036acfa65217d08613d98cd9c86ae46c00

Reply:
Thanks!